yarn.lock

```diff
@@ -3569,9 +3569,9 @@
     csstype "^3.0.2"
 
 "@types/superagent@*":
-  version "4.1.15"
```

Comment:
did we run `yarn bootstrap` to automatically generate these changes?

Reply:
Yes. I run `yarn bootstrap` and yarn.lock has some auto updates